src/coreComponents/physicsSolvers/solidMechanics/benchmarks/DeviatedElasticWellbore.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" ?>
```

Comment:
I suggest to divide it in two xml files. The base one with mesh and some common aspects and the main one with constitutive settings. Then, by simply modifying the mesh of the base file, it can be added to the integrated test.

Reply:
Good idea! However, there is no example of mixing between benchmark and integrated test yet. And as you known I cannot setup integrated test from Pangea. So I would like to skip this.